plugins/catalog/src/components/AboutCard/AboutContent.tsx

```diff
@@ -14,207 +14,42 @@
+import { Grid } from '@material-ui/core';
 import React from 'react';
-import { AboutField } from './AboutField';
-import { LinksGridList } from '../EntityLinksCard/LinksGridList';
-
-const useStyles = makeStyles({
-  description: {
-    wordBreak: 'break-word',
-  },
-});
+import { AboutCardDefaultFields } from './fields';
 
 /**
  * Props for {@link AboutContent}.
  *
  * @public
  */
 export interface AboutContentProps {
-  entity: Entity;
-}
-
-function getLocationTargetHref(
-  target: string,
-  type: string,
-  entitySourceLocation: {
-    type: string;
-    target: string;
-  },
-): string {
-  if (type === 'url' || target.includes('://')) {
-    return target;
-  }
-
-  const srcLocationUrl =
-    entitySourceLocation.type === 'file'
-      ? `file://${entitySourceLocation.target}`
-      : entitySourceLocation.target;
-
-  if (type === 'file' || entitySourceLocation.type === 'file') {
-    return new URL(target, srcLocationUrl).href;
-  }
-
-  return srcLocationUrl;
+  /**
+   * The entity here is for backward compatibility only.
+   * It's not passed to the children as the children use
+   * useEntity hook to access the entity in the current context.
+   *
+   * The {@link AboutContent} component should only be used
+   * within EntityLayout, and the entity prop will be removed
+   * in future releases.
+   *
+   * @deprecated use useEntity hook in children.
+   */
+  entity?: Entity;
+  children?: React.ReactNode;
 }
 
+// TODO: It seems the whole AboutContent component here is redundant,
+// consider deprecating and removing it.
 /** @public */
-export function AboutContent(props: AboutContentProps) {
-  const { entity } = props;
-  const classes = useStyles();
-  const isSystem = entity.kind.toLocaleLowerCase('en-US') === 'system';
-  const isResource = entity.kind.toLocaleLowerCase('en-US') === 'resource';
-  const isComponent = entity.kind.toLocaleLowerCase('en-US') === 'component';
-  const isAPI = entity.kind.toLocaleLowerCase('en-US') === 'api';
-  const isTemplate = entity.kind.toLocaleLowerCase('en-US') === 'template';
-  const isLocation = entity.kind.toLocaleLowerCase('en-US') === 'location';
-  const isGroup = entity.kind.toLocaleLowerCase('en-US') === 'group';
-
-  const partOfSystemRelations = getEntityRelations(entity, RELATION_PART_OF, {
-    kind: 'system',
-  });
-  const partOfComponentRelations = getEntityRelations(
-    entity,
-    RELATION_PART_OF,
-    {
-      kind: 'component',
-    },
-  );
-  const partOfDomainRelations = getEntityRelations(entity, RELATION_PART_OF, {
-    kind: 'domain',
-  });
-  const ownedByRelations = getEntityRelations(entity, RELATION_OWNED_BY);
-
-  let entitySourceLocation:
-    | {
-        type: string;
-        target: string;
-      }
-    | undefined;
-  try {
-    entitySourceLocation = getEntitySourceLocation(entity);
-  } catch (e) {
-    entitySourceLocation = undefined;
+export const AboutContent = (props: AboutContentProps) => {
```

Comment:
Could have kept this as a function instead of a const, right? The thing with functions is that they actually have a name, and that helps with inspection and debugging